### PR TITLE
Marker class

### PR DESCRIPTION
**Pull request by niccireinking**

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/MarkerServlet.java [hunk 1]
```diff
@@ -11,4 +30,46 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     	String json = gson.toJson(markers);
```
Comment (emto) [inline src/main/java/com/google/codeu/servlets/MarkerServlet.java:31]: looks like indent is a bit off for this file

File: src/main/java/com/google/codeu/servlets/MarkerServlet.java [hunk 2]
```diff
@@ -11,4 +30,46 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+    	List<Marker> markers = new ArrayList<>();
+
+    	DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    	Query query = new Query("Marker");
+    	PreparedQuery results = datastore.prepare(query);
+
+    	for (Entity entity : results.asIterable()) {
+      		double lat = (double) entity.getProperty("lat");
+      		double lng = (double) entity.getProperty("lng");
+      		String content = (String) entity.getProperty("content");
```
Comment (emto) [inline src/main/java/com/google/codeu/servlets/MarkerServlet.java:63]: indent like
```
@Override
public void ... () {
    stuff inside
}
```

**Review by miaboloix — APPROVED** [review]
fix indentation then merge!

**Review by Rachelware — APPROVED** [review]
merging because the requests were fixed and to test

**Pull request merged**